mindsdb/integrations/handlers/spacy_handler/requirements.txt

```diff
@@ -0,0 +1,2 @@
+spacy
+en_core_web_sm
```

Comment:
`pip` fails to resolve for this package, and looking at PyPI am not sure it exists. Can you double check this is the right way?

Reply:
My bad! Basically _"en_core_web_sm"_ refers to English language model and to download them you have to use following command: 
_python -m spacy download en_core_web_sm_ 

For others like Finnish or Japan: 
_python -m spacy download fi_core_news_sm_
_python -m spacy download ja_core_news_sm_

I tested 2 days ago with only downloading _"spacy"_ and it seems that English model is included. I do not know whether other language models are.

I will delete _"en_core_web_sm"_ from the _"requirements.txt"_.